Docker/centralserver/README.md

```diff
@@ -0,0 +1,16 @@
+# Central Server Docker Image
+
+The Central Server Docker image contains vanilla X-Road Central Server version 6.20.0 or later.
+All services, including TEST-CA, TSA, OCSP and PostgreSQL database, are installed into the same container and run using supervisord.
```

Comment:
May you please share more info about `run using supervisord`?
I am wondering what is supervisord.

Reply:
A process control system, similar to systemd. Aforementioned services can be run using supervisord, e.g. "supervisorctl start <service_name>"